### ISSUE TITLE
they are banning today??

### ISSUE THREAD

**Issue by webmotril** [PokemonGoF/PokemonGo-Bot#5815]
As the API changed or what's the problem? they have hard ban 4 of my accounts. I don't understand.
Anybody in my case??

**Comment by mmixx** [comment]
Yep... I'm guessing they are... Mine got banned too...

**Comment by VBQL** [comment]
yes the api changed from 0.43 to 0.45, there was a 5 day or so period when this bot had the old api. That is why you should always pay attention to the issues tab.

**Comment by db-coder** [comment]
2 accounts of mine got banned :'(

**Comment by Behemoth8u** [comment]
Mine says probably permabanned when trying to start the bot but works on phone.

**Comment by moquette** [comment]
Banned!

**Comment by FernandoMerino** [comment]
The API was changed from 0.43 to 0.45 and was forced. The bot API was changed to 0.45 yesterday. If you have runned the bot with 0.43 after the forced update, this was the problem.

**Comment by reigelgallarde** [comment]
might be with your settings... or I'm yet to be banned? lol

**Comment by PokemonGoZ** [comment]
Banwaves incoming. Got the heads up from /r/PokemonGoDev

**Comment by takenek** [comment]
Got ban on 2 accounts too.

**Comment by MZorzy** [comment]
on reddit look like the banned bot used for mapping. level 1 with lot of km/miles.
my bot lev 12 & 16 run fine. (sgratt sgratt)
ever use gps altitude (min max default) on command line

**Comment by AWSadmin** [comment]
3 accounts were banned last night. All 22-23 levels, roaming locally.

**Comment by Jcolomar** [comment]
sniping, to much time bot on, through buildings, go faster than human walk speed.... make niantic more easy detect bots

**Comment by paganoll** [comment]
No ban here. 5 bots 10/20h a day. Snipìng always (dragon, snorlax, lapras only) but **not yesterday evening** , From same city, all at same time. Levels among 27-35.  As Lucky as on the last ban...
No gym. No phone app. Only for to envy and astonish nephews ;)

**Comment by solderzzc** [comment]
You should only run dummy accounts during API updating time frame. I have same situation with @paganoll. And I also run it yesterday evening. You need add random alive pause, slow down your speed, skip some trash pokemons instead of capture all you meet.
The most important thing is, **you should only run dummy accounts during API updating time frame**.

**Comment by herbcarroll** [comment]
just discovered two accounts of mine were prompted with captchas when logging in after discovering the script wasn't doing anything.  Nothing banned though.

**Comment by rbusquet** [comment]
Just got the magnemite for captcha check. No ban, yet.

**Comment by MerlionRock** [comment]
New update coming, suggest stay low until API updated to 0.47 API. No force update yet. But if you want to be safe and not being caught during force API update, you might want to stop botting temporary.

**Comment by xiatianyu** [comment]
> New update coming, suggest stay low until API updated to 0.47 API. No force update yet.

Well, that has just happened… A forced update with API version 0.47.1 has landed. Better stop botting for now until the new API becomes integrated.

**Comment by reigelgallarde** [comment]
![image](https://cloud.githubusercontent.com/assets/10612336/20470540/d080b5e2-afec-11e6-8ecf-eecc9864e0c4.png)
